src/main/java/com/blackducksoftware/integration/jira/common/BlackDuckDataHelper.java

```diff
@@ -180,31 +179,20 @@ private int getSumOfRiskCounts(final List<RiskCountView> vulnerabilityCounts) {
         return null;
     }
 
-    public String getFirstLink(final HubView view, final String linkKey) throws IntegrationException {
-        return blackDuckService.getFirstLink(view, linkKey);
+    public Optional<String> getFirstLink(final BlackDuckView view, final String linkKey) {
+        return view.getFirstLink(linkKey);
     }
 
-    public String getFirstLinkSafely(final HubView view, final String linkKey) {
-        String link = null;
-        try {
-            link = blackDuckService.getFirstLink(view, linkKey);
-        } catch (HubIntegrationException e) {
-            logger.debug("Could not get link " + linkKey + " for view: " + view, e);
-        }
-        return link;
+    public String getFirstLinkSafely(final BlackDuckView view, final String linkKey) {
+        return getFirstLink(view, linkKey).orElse(null);
     }
 
-    public String getHref(final HubView view) throws IntegrationException {
-        return blackDuckService.getHref(view);
+    public Optional<String> getHref(final BlackDuckView view) {
```

Comment:
Now that the view itself contains the method to get the link, there is no need to wrap this.